src/pretix/control/templates/pretixcontrol/event/settings.html

```diff
@@ -9,6 +9,7 @@
             <legend>{% trans "General information" %}</legend>
             {% bootstrap_field form.name layout="horizontal" %}
             {% bootstrap_field form.slug layout="horizontal" %}
+            {% bootstrap_field form.location layout="horizontal" %}
```

Comment:
I would recommend moving this two lines down below the date, as you did in the other form, as it is "less relevant".